metaci/testresults/robot_importer.py

```diff
@@ -326,5 +339,5 @@ def export_robot_test_results(flowtest) -> None:
         )
         return
```

Comment:
This isn't a critique on the code, it's just me thinking out loud:

`result` should have an "Accepted" or "Rejected" record for every test case. I'm thinking we should probably log the rejected items so we can answer the inevitable question "why isn't my test showing up in testmanager"? I think if we do a good enough job we should never have tests rejected, since they would typically be rejected only if  we pass invalid data to testmanager.

Reply:
The `return` line here does nothing since this if/else block is the end of the function